converter/proto_json_payload_converter.go

```diff
@@ -41,6 +41,11 @@ import (
 type ProtoJSONPayloadConverter struct {
 	gogoMarshaler   gogojsonpb.Marshaler
 	gogoUnmarshaler gogojsonpb.Unmarshaler
+	options         ProtoJSONPayloadConverterOptions
+}
+
+type ProtoJSONPayloadConverterOptions struct {
```

Comment:
All exported items should have documentation that starts with its name, e.g. `// ProtoJSONPayloadConverterOptions represents options for `NewProtoJSONPayloadConverterWithOptions`.